core/node/node_framework/src/implementations/resources/via_inscriber.rs

```diff
@@ -0,0 +1,15 @@
+// use via_btc_client::inscriber::Inscriber;
+
+use crate::resource::Resource;
+
+/// Represents a client of a certain DA solution.
+#[derive(Debug, Clone)]
+
+// todo add trait to inscriber and then add it as resource in here
+pub struct InscriberResource();
```

Comment:
why do we need resource? If it's the main idea for changing the `via_btc_client` crate, I'm not quite sure about this

Reply:
This code was implemented before, but I think the idea was to keep the same code implemntation approach used by zkSync team. In the last implementation it was removed.